docs/development/how-to-maintain.rst

```diff
@@ -0,0 +1,109 @@
+How To Maintain
+===============
+
+This document is dedicated to maintainers of respy.
+
+
+Versioning
+----------
+
+respy adheres in large parts to `semantic versioning <https://semver.org>`_. Thus, for a
+given version number ``major.minor.patch``
+
+* ``major`` is incremented when you make incompatible API changes.
+* ``minor`` is incremented when you add functionality which is backwards compatible.
+* ``patch`` is incremented when you make backwards compatible bug fixes.
+
+Branching Model
+---------------
+
+The branching model for respy is very simple.
```

Comment:
``respy``, probably good to do a search through the whole doc to make sure we have it all aligned.